apkid/rules/elf/protectors.yara

```diff
@@ -162,3 +162,45 @@ rule vkey_elf : protector
   condition:
     is_elf and $libname and 1 of ($vos*) and 1 of ($detection*) and 1 of ($jni*)
 }
+
+rule verimatrix_elf_arm64 : protector
+{
+  meta:
+    description = "InsideSecure Verymatrix"
+    url         = "https://www.verimatrix.com/products/app-shield/"
+    sample      = "105a2646e8acf45afcd606f4d47af68a8218e0ae7d8c9646d0048cbf32df7a73"
+    author      = "FrenchYeti"
+    
+  strings:
+    // byte sequence from .data, used into JNI_OnLoad
+    $seq = {
+      ?? ?? ?? ?? 88 98 a8 b8
```

Comment:
Any useful comment on the side that we can include?

Reply:
Unfortunately not

Reply:
Have you found samples with this `.data` fingerprint (or watermarking)? If its unique per developer, it won't match other developer' SDKs. Right?

Reply:
Found in several samples (totally more than 20), including 11 from Koodous  [https://koodous.com/rulesets/8164](https://koodous.com/rulesets/8164)

Reply:
OK. Perfect